### ISSUE TITLE
[DataCap Application] <Suzhou Qudoufeng Network Technology Co., Ltd.> 

### ISSUE THREAD

**Issue by Qudoufeng** [filecoin-project/filecoin-plus-large-datasets#638]
# Large Dataset Notary Application

To apply for DataCap to onboard your dataset to Filecoin, please fill out the following.

## Core Information
- Organization Name: Suzhou Qudoufeng Network Technology Co., Ltd.
- Website / Social Media:www.aicarmap.com
- Total amount of DataCap being requested (between 500 TiB and 5 PiB):5PiB
- Weekly allocation of DataCap requested (usually between 1-100TiB):200TiB
- On-chain address for first allocation:f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq

_Please respond to the questions below by replacing the text saying "Please answer here". Include as much detail as you can in your answer._

## Project details

Share a brief history of your project and organization.
```
Suzhou Qudoufeng Network Technology Co., Ltd., developed the Qudoufeng APP, which is the first real-time LBS self-driving tour guide tool in China, and is committed to creating a more convenient and interesting self-driving travel experience. Provide a new way of self-driving travel, so that self-driving strategies are no longer limited to text and pictures, and create an era of intelligent self-driving travel.
Qudoufeng mainly provides users with intelligent self driving travel services. Therefore, many factors including route, weather, accommodation, scenic spots and environment need to be considered. The remote sensing images in the sample data are machine learning sample data of weather and route. Through these data, artificial intelligence programs are trained to achieve business intelligence.
```

What is the primary source of funding for this project?
```
Financing and Operating Income.
```

What other projects/ecosystem stakeholders is this project associated with?
```
No.
```

## Use-case details

Describe the data being stored onto Filecoin
```
Remote sensing images / map mapping / map data / big data analysis.
```

Where was the data in this dataset sourced from?
```
The data comes from our company's R&D department, technology department, etc. The above data are mainly presented in the form of pictures and documents.
```

Can you share a sample of the data? A link to a file, an image, a table, etc., are good ways to do this. 
```
https://drive.google.com/drive/folders/1ExROmt61gRErrEqzSAZyYKmMvF4l_y54?usp=sharing
```
        
Confirm that this is a public dataset that can be retrieved by anyone on the Network (i.e., no specific permissions or access rights are required to view the data).
```
Yes.
```

What is the expected retrieval frequency for this data?
```
It can be retrieved at any time.
```

For how long do you plan to keep this dataset stored on Filecoin?
```
Three years or more.
```

## DataCap allocation plan

In which geographies (countries, regions) do you plan on making storage deals?
```
In China or Asia.
```

How will you be distributing your data to storage providers? Is there an offline data transfer process?
```
Mainly through online transmission, if the storage provider is in China, it can be transmitted offline.
```

How do you plan on choosing the storage providers with whom you will be making deals? This should include a plan to ensure the data is retrievable in the future both by you and others.
```
We will screen reliable storage providers through various channels to ensure that data can be stored for a long time and can be retrieved at any time.
```

How will you be distributing deals across storage providers?
```
We will distribute them equally to the high-quality storage providers that pass our audit.
```

Do you have the resources/funding to start making deals as soon as you receive DataCap? What support from the community would help you onboard onto Filecoin?
```
The company has provided a special fund for this matter to cover all expenses. If we need help in the future, we will consult you again.
```

**Comment by Qudoufeng** [comment]
1. In China, we currently contact 5 SPS, and their nodes are
f01782338
f01741924
f0469055
In Hong Kong, we contacted one sp,
f01611281
In Singapore, we contacted one sp,
f01402814
We also plan to find some high-quality SPS with the help of BDE platform and notaries

————————
The nodes listed above are not necessarily the final ones. We are trying to contact SPS in more regions

2. We have sent a verification email, please check it.

![62acd1fe71e646f64eade7d3873a75f](https://user-images.githubusercontent.com/111046445/187820919-c88336bf-44b5-439b-9cee-e1c4d7605825.jpg)

**Comment by Sunnyiscoming** [comment]
Received that.

**Comment by raghavrmadya** [comment]
Hi, do you also have video data? 5 PiBs of images is a little hard to wrap our heads around. Any additional data samples and evidence supporting the requested amounts would be great.

**Comment by Qudoufeng** [comment]
Suzhou Qudoufeng Network Technology Co., Ltd., developed the Qudoufeng APP, which is the first real-time LBS self-driving tour guide tool in China.
Qudoufeng mainly provides users with intelligent self driving travel services. Therefore, many factors including route, weather, accommodation, scenic spots and environment need to be considered. The remote sensing images in the sample data are machine learning sample data of weather and route. Through these data, artificial intelligence programs are trained to achieve business intelligence.

The data set is mainly remote sensing data
Remote sensing image file content
 1. TIF imgae file （90%）
    ①B_.TIF: remote sensing image
    ②BQA.TIF: image quality information 
 2. TXT text document （10%）
    ①ANG: projection information
    ②MTL: image metadata (header file)

As shown in the figure，The total number of entries of the remote sensing data set is 1584081

![image](https://user-images.githubusercontent.com/111046445/189065967-0f893108-69c1-49e9-bc4e-9e28993ca724.png)

The telemetry data content corresponding to each entry ranges from 1.7GiB to 2.5GiB.

![image](https://user-images.githubusercontent.com/111046445/189066235-a2ac192c-5fb5-4934-93d3-83084455c806.png)
![image](https://user-images.githubusercontent.com/111046445/189066335-55e56263-b132-4abc-a077-80f3dc0e76a1.png)
![image](https://user-images.githubusercontent.com/111046445/189066397-bea8658c-6f03-4f3f-ab91-c4307990d299.png)


 In short, our dataset has
1584081 * 1.5GiB ～ 2GiB ≈ 2.26PIB - 3PIB
Because this is the first time we know about filecoin, we plan to save 1 / 3 of the data set in filecoin first.This is about 1PIB of data, and we plan to make 6 replicas around the world.

**Comment by raghavrmadya** [comment]
### Datacap Request Trigger
**Total DataCap requested**

> 5PiB

**Expected weekly DataCap usage rate**

> 200TiB

**Client address**

>f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq

**Comment by raghavrmadya** [comment]
### Datacap Request Trigger
**Total DataCap requested**

> 5PiB

**Expected weekly DataCap usage rate**

> 200TiB

**Client address**

>f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacect5idmuam7lvtho4tx2jybxy7xskauurt2c7p3fjkthhlrzi6yby 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacect5idmuam7lvtho4tx2jybxy7xskauurt2c7p3fjkthhlrzi6yby

**Comment by stcouldlisa** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedl42wloaikkhc7fvv3fg4ivryypes3wuymhy5plbrsaeyrhumg2a 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 100.00TiB
#### Signer Address
> f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedl42wloaikkhc7fvv3fg4ivryypes3wuymhy5plbrsaeyrhumg2a

**Comment by BigD-Exchange** [comment]
@Qudoufeng Hi! Great to see you have gotten approval for DataCap and advancing the mission of preserving humanity’s most important information. If you are looking for storage providers to store these data or have any questions, please visit #bigdata-exchange on Filecoin Slack or reply here. 

We have strong demand from a diverse group of SPs, who are actively looking to onboard more data.

**Comment by Tom-OriginStorage** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebqa2364xdqiebsfqfx7gkbemjdn5pjh3tqca72l6bfiqtg36uzcs 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebqa2364xdqiebsfqfx7gkbemjdn5pjh3tqca72l6bfiqtg36uzcs

**Comment by kernelogic** [comment]
Client has informed me will pay close attention to distribution ratio next time (this round 100% is operation error). Will keep close monitor to this.

**Comment by kernelogic** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceblis7caykml4ohkdcn6b6whvezhkr7nfzklik3y5ooxcih4ff242 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 200.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceblis7caykml4ohkdcn6b6whvezhkr7nfzklik3y5ooxcih4ff242

**Comment by newwebgroup** [comment]
The client contacted us and said that because of the COVID-19 in China, some SPs could not receive the express delivery of Dataset, which has been resolved.
In the latest round, the storage ratio will be adjusted.
I'll keep an eye on it.

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceav2dvxxdchjijtptouo3tw4hrxxcemqwu5ea26362b7rijpji5jc 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### Id
> baad3bf7-b470-49db-9efd-05024d33e346
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceav2dvxxdchjijtptouo3tw4hrxxcemqwu5ea26362b7rijpji5jc

**Comment by NDLABS-OFFICE** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacealbau7zcflobekxm4bl5yitjbprjpjtmgw3t576q327szx7mxkdi 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 400.00TiB
#### Signer Address
> f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
#### Id
> baad3bf7-b470-49db-9efd-05024d33e346
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacealbau7zcflobekxm4bl5yitjbprjpjtmgw3t576q327szx7mxkdi

**Comment by Tom-OriginStorage** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecokt26qu3vneaocoyxj6zpovvyc4lem36rhwuvt5ngvuzlsley72 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### Id
> f0fc2161-dde1-4253-83ad-e82b5eba3673
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecokt26qu3vneaocoyxj6zpovvyc4lem36rhwuvt5ngvuzlsley72

**Comment by stcouldlisa** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceb2jt2tnjzvb242h26ilmm7er7vcf5fdlo7w7pcwj4sbwxuanizq4 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 800.00TiB
#### Signer Address
> f1jvvltduw35u6inn5tr4nfualyd42bh3vjtylgci
#### Id
> f0fc2161-dde1-4253-83ad-e82b5eba3673
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceb2jt2tnjzvb242h26ilmm7er7vcf5fdlo7w7pcwj4sbwxuanizq4

**Comment by stcouldlisa** [comment]
<img width="1371" alt="WechatIMG2386" src="https://user-images.githubusercontent.com/112812896/203688014-df04d388-8508-4445-905a-73732cd45009.png">

**Comment by stcouldlisa** [comment]
I checked the situation of this client, the allocation is reasonable, and I have contacted many SPs, so I am willing to sign for him

**Comment by stcouldlisa** [comment]
I checked the situation of this client, the allocation is reasonable, and  have contacted many SPs, so I am willing to sign for him

**Comment by newwebgroup** [comment]
The following two items were checked
1. DC distribution proportion
2. Location distribution of SPs
All meet the requirements

<img width="1606" alt="image" src="https://user-images.githubusercontent.com/100118145/204736495-de6ce8a7-9d1a-4c37-9170-9217cb248201.png">

**Comment by newwebgroup** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacea56odnxmkwflk7zgonbo2xobmxzsfzpkropm4zx3wkrcdhon2bp6 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 1.56PiB
#### Signer Address
> f1e77zuityhvvw6u2t6tb5qlnsegy2s67qs4lbbbq
#### Id
> c6f8d9a4-5633-4cb8-b39e-14c2b0521e22
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacea56odnxmkwflk7zgonbo2xobmxzsfzpkropm4zx3wkrcdhon2bp6

**Comment by liyunzhi-666** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecd2s4dsfgezm6nku23ipv4buplfm37mthkujezqavyt2mf7fbey4 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 1.56PiB
#### Signer Address
> f1pszcrsciyixyuxxukkvtazcokexbn54amf7gvoq
#### Id
> c6f8d9a4-5633-4cb8-b39e-14c2b0521e22
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecd2s4dsfgezm6nku23ipv4buplfm37mthkujezqavyt2mf7fbey4

**Comment by ipfscn** [comment]
The customer's node allocation ratio is in line with the FIL+ rule

**Comment by 1ane-1** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaa4ho5weabeiuiw5m47uyf6nqeel6livscfuvpzjy3xexvdkijjm 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 1.56PiB
#### Signer Address
> f1mdk7s2vntzm6hu35yuo6vjubtrpfnb2awhgvrri
#### Id
> ececafa3-cfe1-44b0-ac9b-504888edfc1d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaa4ho5weabeiuiw5m47uyf6nqeel6livscfuvpzjy3xexvdkijjm

**Comment by filplus-checker** [comment]
## DataCap and CID Checker Report[^1]
 - Organization: `Suzhou Qudoufeng Network Technology Co., Ltd.`
 - Client: `f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq`
### Storage Provider Distribution
The below table shows the distribution of storage providers that have stored data for this client.

If this is the first time a provider takes verified deal, it will be marked as `new`.

For most of the datacap application, below restrictions should apply.
 - Storage provider should not exceed 25% of total datacap.
 - Storage provider should not be storing duplicate data for more than 20%.
 - Storage provider should have published its public IP address.
 - All storage providers should be located in different regions.

⚠️ 60.60% of total deal sealed by [f01981603](https://filfox.info/en/address/f01981603) are duplicate data.

⚠️ [f01981603](https://filfox.info/en/address/f01981603) has unknown IP location.

⚠️ 43.71% of total deal sealed by [f01919535](https://filfox.info/en/address/f01919535) are duplicate data.

⚠️ [f01964215](https://filfox.info/en/address/f01964215) has unknown IP location.

⚠️ [f01959805](https://filfox.info/en/address/f01959805) has unknown IP location.

⚠️ 53.61% of total deal sealed by [f01566485](https://filfox.info/en/address/f01566485) are duplicate data.

⚠️ 73.52% of total deal sealed by [f01699999](https://filfox.info/en/address/f01699999) are duplicate data.

⚠️ 44.73% of total deal sealed by [f01876488](https://filfox.info/en/address/f01876488) are duplicate data.

⚠️ 41.24% of total deal sealed by [f01953959](https://filfox.info/en/address/f01953959) are duplicate data.

⚠️ 33.22% of total deal sealed by [f01943316](https://filfox.info/en/address/f01943316) are duplicate data.

⚠️ 60.20% of total deal sealed by [f01920091](https://filfox.info/en/address/f01920091) are duplicate data.

| Provider                                                    |                           Location | Total Deals Sealed | Percentage | Unique Data | Duplicate Deals |
| :---------------------------------------------------------- | ---------------------------------: | -----------------: | ---------: | ----------: | --------------: |
| [f01981603](https://filfox.info/en/address/f01981603)`new`  |                            Unknown |         872.38 TiB |     23.39% |  343.75 TiB |          60.60% |
| [f01919535](https://filfox.info/en/address/f01919535)       |                   Wuhan, Hubei, CN |         604.78 TiB |     16.22% |  340.44 TiB |          43.71% |
| [f01964215](https://filfox.info/en/address/f01964215)`new`  |                            Unknown |         388.84 TiB |     10.43% |  326.38 TiB |          16.07% |
| [f01959805](https://filfox.info/en/address/f01959805)`new`  |                            Unknown |         288.56 TiB |      7.74% |  288.47 TiB |           0.03% |
| [f01566485](https://filfox.info/en/address/f01566485)       |            Shenzhen, Guangdong, CN |         269.75 TiB |      7.23% |  125.13 TiB |          53.61% |
| [f01699999](https://filfox.info/en/address/f01699999)       |           Zhongshan, Guangdong, CN |         241.66 TiB |      6.48% |   64.00 TiB |          73.52% |
| [f01876488](https://filfox.info/en/address/f01876488)       |  Ho Chi Minh City, Ho Chi Minh, VN |         225.81 TiB |      6.06% |  124.81 TiB |          44.73% |
| [f01953959](https://filfox.info/en/address/f01953959)       |           Singapore, Singapore, SG |         204.00 TiB |      5.47% |  119.88 TiB |          41.24% |
| [f01953944](https://filfox.info/en/address/f01953944)       |           Singapore, Singapore, SG |         109.88 TiB |      2.95% |  106.75 TiB |           2.84% |
| [f01777785](https://filfox.info/en/address/f01777785)       |           Singapore, Singapore, SG |         108.53 TiB |      2.91% |   94.50 TiB |          12.93% |
| [f01777786](https://filfox.info/en/address/f01777786)       |                    Ōi, Saitama, JP |          81.56 TiB |      2.19% |   78.09 TiB |           4.25% |
| [f01943316](https://filfox.info/en/address/f01943316)       |         Piscataway, New Jersey, US |          75.16 TiB |      2.02% |   50.19 TiB |          33.22% |
| [f01777788](https://filfox.info/en/address/f01777788)       | Hong Kong, Central and Western, HK |          72.66 TiB |      1.95% |   72.66 TiB |           0.00% |
| [f01920091](https://filfox.info/en/address/f01920091)       |       Frankfurt am Main, Hesse, DE |          59.44 TiB |      1.59% |   23.66 TiB |          60.20% |
| [f01527777](https://filfox.info/en/address/f01527777)`new`  |               Nanning, Guangxi, CN |          47.13 TiB |      1.26% |   47.13 TiB |           0.00% |
| [f01520487](https://filfox.info/en/address/f01520487)`new`  |                 Xiamen, Fujian, CN |          46.69 TiB |      1.25% |   46.69 TiB |           0.00% |
| [f01777787](https://filfox.info/en/address/f01777787)       |                Paripark, Seoul, KR |          23.75 TiB |      0.64% |   23.75 TiB |           0.00% |
| [f01529999](https://filfox.info/en/address/f01529999)`new`  | Hong Kong, Central and Western, HK |           8.69 TiB |      0.23% |    8.69 TiB |           0.00% |

![Provider Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/638/1671097585799.png)
### Deal Data Replication
The below table shows how each many unique data are replicated across storage providers.
- No more than 25% of unique data are stored with less than 4 providers.

✔️ Data replication looks healthy.

| Unique Data Size | Total Deals Made | Number of Providers | Deal Percentage |
| ---------------: | ---------------: | ------------------: | --------------: |
|        26.84 TiB |        26.84 TiB |                   1 |           0.72% |
|       800.00 GiB |         3.59 TiB |                   2 |           0.10% |
|        13.06 TiB |        75.13 TiB |                   3 |           2.01% |
|        80.69 TiB |       575.75 TiB |                   4 |          15.44% |
|        65.72 TiB |       577.69 TiB |                   5 |          15.49% |
|        38.28 TiB |       368.28 TiB |                   6 |           9.88% |
|        46.13 TiB |       512.47 TiB |                   7 |          13.74% |
|        19.88 TiB |       245.78 TiB |                   8 |           6.59% |
|        12.78 TiB |       174.25 TiB |                   9 |           4.67% |
|        21.50 TiB |       330.22 TiB |                  10 |           8.85% |
|        26.22 TiB |       453.66 TiB |                  11 |          12.16% |
|        17.50 TiB |       344.25 TiB |                  12 |           9.23% |
|         2.00 TiB |        41.34 TiB |                  13 |           1.11% |

![Replication Distribution](https://raw.githubusercontent.com/data-preservation-programs/filplus-checker-assets/main/filecoin-project/filecoin-plus-large-datasets/issues/638/1671097586316.png)
### Deal Data Shared with other Clients
The below table shows how many unique data are shared with other clients.
Usually different applications owns different data and should not resolve to the same CID.

⚠️ CID sharing has been observed.

| Other Client                                                                                                                                                                                                              | Application          | Total Deals Affected | Unique CIDs |          Verifier |
| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------- | -------------------: | ----------: | ----------------: |
| [f3srsn5u2owt5vwvuaq6ltelr4fecqz5qptfbnj2p<br/>lbv5q5aowhyf4vonbzuxcswp5owuygb4u27ozox5b<br/>ppfa](https://filfox.info/en/address/f3srsn5u2owt5vwvuaq6ltelr4fecqz5qptfbnj2plbv5q5aowhyf4vonbzuxcswp5owuygb4u27ozox5bppfa) | `Glif auto verified` |           256.00 GiB |           1 | Jonathan Schwartz |

[^1]: To manually trigger this report, add a comment with text `checker:manualTrigger`

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaced4s7fde2qpaqfvxbgludtznxox44fzsl66u6xy4q4t2gpnsczlh2 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 425.11TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> a2b345e4-ac47-4981-9e7d-0eff9993050d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaced4s7fde2qpaqfvxbgludtznxox44fzsl66u6xy4q4t2gpnsczlh2

**Comment by kernelogic** [comment]
CID Checker report looks alright, some duplications can be tolerated for additional replication.

**Comment by Tom-OriginStorage** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebpaxtxddbkag6t5ds2cvqdfryu23koifrlx7fek2cfgxd3b7zrss 
#### Address 
> f1gk7n4ckn23ix2bu7q6qtas6eayambn57ass2zfq
#### Datacap Allocated
> 425.11TiB
#### Signer Address
> f1q6bpjlqia6iemqbrdaxr2uehrhpvoju3qh4lpga
#### Id
> a2b345e4-ac47-4981-9e7d-0eff9993050d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebpaxtxddbkag6t5ds2cvqdfryu23koifrlx7fek2cfgxd3b7zrss

**Comment by cryptowhizzard** [comment]
Hi there,

I have downloaded some data but it seems you are storing something else then what you have requested datacap for.

baga6ea4seaqoq2y6dulzoedywhp2l3fxlrqulil7gtdnink2izqzfe2fgysimgq.car

<img width="546" alt="Scherm­afbeelding 2023-02-06 om 21 53 41" src="https://user-images.githubusercontent.com/36404435/217083207-413c87f3-cc2e-48ce-ae97-aac3a3c4ec0a.png">

**Comment by NDLABS-OFFICE** [comment]
@cryptowhizzard excellent work. 
And ND wait for the client to reply.

**Comment by Qudoufeng** [comment]
Hello，Sir. @cryptowhizzard 
Through our technical staff check, found that this CID does not belong to our data set
This is all the records we found about this CID. 
Please correct me if there is any error.

_**“baga6ea4seaqoq2y6dulzoedywhp2l3fxlrqulil7gtdnink2izqzfe2fgysimgq”**_
> 
> {‘Proposal’: {‘PieceCID’: {‘/’: ‘baga6ea4seaqoq2y6dulzoedywhp2l3fxlrqulil7gtdnink2izqzfe2fgysimgq’}, ‘PieceSize’: 34359738368, ‘VerifiedDeal’: True, ‘Client’: ‘f01901637’, ‘Provider’: ‘f01946104’, ‘Label’: ‘QmdryYZBMEZbYo4WHk93hDk5SVT2cxaMuVduvNdL6aC4Th’, ‘StartEpoch’: 2353784, ‘EndEpoch’: 3881175, ‘StoragePricePerEpoch’: ‘0’, ‘ProviderCollateral’: ‘8053340117527284’, ‘ClientCollateral’: ‘0’}, ‘State’: {‘SectorStartEpoch’: 2338483, ‘LastUpdatedEpoch’: 2578999, ‘SlashEpoch’: -1, ‘VerifiedClaim’: 0}}
> {‘Proposal’: {‘PieceCID’: {‘/’: ‘baga6ea4seaqoq2y6dulzoedywhp2l3fxlrqulil7gtdnink2izqzfe2fgysimgq’}, ‘PieceSize’: 34359738368, ‘VerifiedDeal’: True, ‘Client’: ‘f01901637’, ‘Provider’: ‘f01270767’, ‘Label’: ‘QmdryYZBMEZbYo4WHk93hDk5SVT2cxaMuVduvNdL6aC4Th’, ‘StartEpoch’: 2387877, ‘EndEpoch’: 3914604, ‘StoragePricePerEpoch’: ‘0’, ‘ProviderCollateral’: ‘8260939789428882’, ‘ClientCollateral’: ‘0’}, ‘State’: {‘SectorStartEpoch’: 2374307, ‘LastUpdatedEpoch’: 2578783, ‘SlashEpoch’: -1, ‘VerifiedClaim’: 0}}

**Comment by cryptowhizzard** [comment]
Hi,

I don't think so. It still gives me the same pictures.

This is something i received. The f01940441 is your address. f01953944 is in your miners list. 

baga6ea4seaqlm7xoltzrvh6hecs3gt55lpqlwvahjxdqho5hyy7qq7doutnlcka is really there and one of your deals.

The data inside this deal can be viewed here.

http://www.datasetcreators.com/downloadedcarfiles/f01940441-f01953944-baga6ea4seaqlm7xoltzrvh6hecs3gt55lpqlwvahjxdqho5hyy7qq7doutnlcka.car1/
